src/Provider/PointCloudProvider.js

```diff
@@ -236,27 +231,21 @@ export default {
             parseOctree(layer, layer.metadata.hierarchyStepSize, node).then(() => command.view.notifyChange(false));
         }
 
-        const extension = layer.metadata.customBinFormat ? 'cin' : 'bin';
-
         // `isLeaf` is for lopocs and allows the pointcloud server to consider that the current
         // node is the last one, even if we could subdivide even further.
         // It's necessary because lopocs doens't know about the hierarchy (it generates it on the fly
         // when we request .hrc files)
-        const url = `${node.baseurl}/r${node.name}.${extension}?isleaf=${command.isLeaf ? 1 : 0}`;
+        const url = `${node.baseurl}/r${node.name}.${layer.extension}?isleaf=${command.isLeaf ? 1 : 0}`;
 
-        return loadPointFile(layer, url).then((points) => {
+        return Fetcher.arrayBuffer(url, layer.fetchOptions).then(layer.parse).then((geometry) => {
+            const points = new THREE.Points(geometry, layer.material.clone());
+            addPickingAttribute(points);
+            points.frustumCulled = false;
+            points.matrixAutoUpdate = false;
             points.position.copy(node.bbox.min);
             points.scale.set(layer.metadata.scale, layer.metadata.scale, layer.metadata.scale);
-            points.tightbbox.min.x *= layer.metadata.scale;
-            points.tightbbox.min.y *= layer.metadata.scale;
-            points.tightbbox.min.z *= layer.metadata.scale;
-            points.tightbbox.max.x *= layer.metadata.scale;
-            points.tightbbox.max.y *= layer.metadata.scale;
-            points.tightbbox.max.z *= layer.metadata.scale;
-            points.tightbbox.translate(node.bbox.min);
-            points.material.transparent = layer.opacity < 1.0;
-            points.material.uniforms.opacity.value = layer.opacity;
```

Comment:
material update is now triggered in the Process update (previously transparency was not updated !)